### DISCUSSION TITLE
OpenBao Marketing & Publicity

### DISCUSSION THREAD

**Discussion by joewxboy** [openbao/openbao#148] [Ideas]
I'd like to begin a discussion around a plan for communicating what the OpenBao project is accomplishing.  These can be considered official announcements, and as such, likely require input/approval from TSC members.  Here's a tentative plan for topics and a publishing cadence to keep up a steady drumbeat.

-----

## Proposed Topics

Are these good topics to discuss?  What questions should they answer?  What are key points we want to communicate?

1. Open Horizon has been incubating OpenBao since November 7, 2023. OpenBao has now reach sufficient maturity with the creation of a project Technical Steering Committee (TSC) and the nomination of a TSC Chair, Nathan Phelps. The TSC voting members are comprised of the initial five founding companies: IBM (IBM Software), IOTech Systems, Zededa, Viaccess-Orca (Orange), and Wallix.
2. The OpenBao TSC has voted to apply to the Linux Foundation’s LF Edge so that the project will fall under their open governance. The codebase will be contributed to LF Edge. Nathan Phelps, Chair of OpenBao, has presented a project proposal to the LF Edge TAC, which will now begin the review and voting process.
3. OpenBao has created a preliminary release timeline, beginning with an Alpha release by the end of March. Statement about API compatibility, any new features, how to work with the project.
4. LF Edge has voted to approve the OpenBao project proposal and OpenBao will become the newest LF Edge Stage One project. With this event, OpenBao has officially exited incubation from the Open Horizon project.
5. OpenBao will have a public demonstration of their Alpha release at ONE Summit in the IBM and LF Edge booths.

## Publishing cadence

Every 14 days (two weeks) would be a preferred starting point, beginning the week of February 16th.  Does that sound like a reasonable schedule?  Are there specific days of the week that would be preferable for publishing?

## Publishing outlets

Where would be a reasonable outlet for hosting the original posts?  For the first one, we can use LF Edge since the article is focused on Open Horizon.  Likewise, the second article has an LF Edge focus, so we might be able to publish there.  The third is iffy, since it doesn't directly focus on LF Edge, and the LF Edge voting period will likely not be concluded.  The fourth and fifth might align with OpenBao being an official standalone project if all goes well.

**Comment by JanMa** [comment]
I like the proposed posts and timeline 👍 
Two questions come to mind: what do you mean with "the codebase will be contributed to LF Edge"? Are we planning to move the repo to their org? And also I was wondering if we can maybe get the website up and running in time to publish the third post? I think announcing an alpha release would be the perfect opportunity to finish it

**Reply by naphelps** [reply]
The repository will not change but source and etc will be transferred to the Linux Foundation.

**Reply by joewxboy** [reply]
Part of the trick with open governance is ensuring that the codebase is safeguarded by a neutral third party ... ideally a foundation.  So when this happens, we'll add LF as the repository owner, and we'll add their admins to the GitHub project.  In turn, they'll provide services for us like DCO checking on all commits and standardization.  They'll additionally help us out with repo admin tasks when we ask for it.  Let's discuss in the community call if you want to dig into this a bit.

**Reply by JanMa** [reply]
Thanks for the explanation :-) I can't promise yet if I'll make it to the next community call(s). They happen at 5pm German time which is terribly inconvenient for me :grin:

**Comment by joewxboy** [comment]
@JanMa so ... a goal of getting the docs website up in about a month?  Sounds good.  Do you have a rough idea of what's left to do?

**Reply by JanMa** [reply]
Might be a bit ambitious but why not :-) I started to write some issues for this yesterday, but I think the most important points are building a proper landing page and buying a domain and maybe reworking the Logo. There's not much left to edit _content_ of the docs apart from replacing the images and diagrams. The missing tutorials can be added later I guess

**Reply by joewxboy** [reply]
OK.  Domain will likely have to wait until we're officially an LF Edge project.
